content/docs/0.14.x/bridge/manage_sequences/index.md

```diff
@@ -65,13 +65,20 @@ Fields:
 link="./assets/trigger-05-trigger-evaluation-timeframe"
 caption="Sequences">}}
 
-#### **Start / end date**
-Please mind, that an evaluation is performed in the past. That means that the start date has to be **after** the end date.
+Special considerations:
+* If you set a timeframe but no start date, the evaluation is performed on the previous timeframe span. This is the only case where evaluations are performed on data from the past.
+* When a start date is specified, the evaluation is performed from this point to an end date. The end date can be specified or is calculated from the timeframe value.
+* Please note: If the timeframe is calculated to an end date too far into the future, evaluation sequences might fail depending on the configured SLI provider.
+
+{{< popup_image
```

Comment:
The images are not displayed in the preview, please double-check: 

![image](https://user-images.githubusercontent.com/729071/163195237-6979cf8e-e94d-4dd6-aada-0bcfe5e94e71.png)